types/src/augment/augment-api-consts.ts

```diff
@@ -585,10 +585,6 @@ declare module '@polkadot/api-base/types/consts' {
        * Exports `Set Councilor Reward Proposal Parameters` proposal parameters.
        **/
       setCouncilorRewardProposalParameters: PalletProposalsEngineProposalParameters & AugmentedConst<ApiType>;
-      /**
-       * Era payout damping factor
-       **/
-      setEraPayoutDampingFactorProposalParameters: PalletProposalsEngineProposalParameters & AugmentedConst<ApiType>;
```

Comment:
Hey @ignazio-bovo this last commit removes the set era payout damping factor proposal types to add the update token pallet governance ones, is it a mistake ?

FYI It also makes the integration tests fail (including the chain setup scenario) so I had to rebase the QN work on the previous commit.

Reply:
Yeah the PR needs to be rebased on luxor and types regenerated.